exonum/Cargo.toml

```diff
@@ -34,11 +34,11 @@ env_logger = "=0.5.12"
 atty = "=0.2.11"
 bytes = "=0.4.9"
 futures = "=0.1.23"
+tokio = "=0.1.7"
```

Comment:
As `tokio` bundles most of the components together, could it make sense to use them through the `tokio` crate? I'm particularly interested in `tokio-core`, [which is deprecated](https://github.com/tokio-rs/tokio-core):

> This crate is scheduled for deprecation in favor of tokio.
>
> tokio-core is still actively maintained, but only bug fixes will be applied. All new feature development is happening in tokio.

Regarding [ECR-740](https://jira.bf.local/browse/ECR-740): I think it could be at least worth trying removing `tokio-core` from our deps; first, because it's deprecated; second, because `tokio` contains [`Runtime` interface](https://tokio-rs.github.io/tokio/tokio/runtime/index.html), which bundles together event loop, thread pool, task spawning, etc., which could make tasks like implementing a thread pool for transactions unnecessary _by design_ (unless we'd like to implement advanced stuff, something like GPU-accelerated verification).